rubocop.test.yml

```diff
@@ -21,3 +21,5 @@ Naming/VariableNumber:
   - 'test/**/*_test.rb' # minitest
   - 'spec/**/*_spec.rb' # RSpec
 
+Metrics/ClassLength:
+  Enabled: false
```

Comment:
Please only exclude test files, just like above.

Reply:
I fixed it and updated the working branch.